src/config/2020.json

```diff
@@ -880,6 +880,16 @@
       "website": "http://www.opentagclosetag.com/",
       "github": "oluoluoxenfree"
     },
+    "alextait1": {
+      "name": "Alex Tait",
```

Comment:
Looks great. Could you move this up to be in sorted order by first name?

Reply:
done :)